test/Spec/State/For.spec.luau

```diff
@@ -171,7 +171,7 @@ return function()
 		local numCalls = 0
 		local forObject = For(scope, data, function(scope, inputPair)
 			numCalls += 1
-			return Computed(scope, function(use)
+			return Computed(scope, function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
			return Computed(scope, function(use)
```

Reply:
reverted